middleware/validationMiddleware.js

```diff
@@ -21,7 +23,22 @@ exports.validateRegistration = [
     }
 ];
 
-exports.validateLogin=[
+exports.validateLogin = [
     body('username').notEmpty().withMessage('Username is required'),
-    body('password').notEmpty().withMessage('Password is required')
+    body('password').notEmpty().withMessage('Password is required').custom(async (password, { req }) => {
```

Comment:
needed a check for password === confirmpassword why we moved the lofin logic here?

Reply:
for register

Reply:
Already mentioned validation for password and confirm password in validateRegistration(line no. 7 to 12).

Reply:
ok but the password match logic should be inside controller move itinto controller ,merge it then move to next question